src/Main.java

```diff
@@ -19,6 +19,8 @@ public static void main(String[] args) {
 
                 switch (choice) {
                     case 1:
+                        SnakesAndLadders obj = new SnakesAndLadders();
+                        obj.snakesAndLadders();
                         break;
 
                     case 2:
```

Comment:
Remove these cases, if not in use or write the comments

Reply:
Done